### ISSUE TITLE
mArchiveList for Spitzer

### ISSUE THREAD

**Issue by siodmiak** [Caltech-IPAC/Montage#9]
Is there a possibility to download IRAC and MIPS images from IRSA with mArchiveList? What is the survey name for it?

Thanks.

**Comment by JohnGood** [comment]
Unofficially, yes (i.e., we haven't fully tested or announced it).  The survey names are IRAC and MIPS and the wavelength bands are either numeric (1,2,3,4 and 1,2,3 respectively) or actual wavelengths (3.6, 4.5, 5.8, 8 and 24, 70, 160).

**Comment by siodmiak** [comment]
Well... it isn't working. I've tried different combination - always get 
"Problem with IRAC file retrieved from IRSA".

Regards,
Natasza


W dniu 2016-12-05 o 14:51, John C Good pisze:

**Comment by JohnGood** [comment]
I just tried it and it worked:

   mArchiveList IRAC 1 m51 0.5 0.5 irac.tbl

returned a list of 61 IRAC 3.6 micron images.  I'm using the next release version of Montage but that shouldn't matter.  Are you using the current GitHub version or an earlier one?

**Comment by siodmiak** [comment]
The current GitHub.

today's attempt:

mArchiveList IRAC 1 m51 0.5 0.5 irac.tbl
[struct stat="ERROR", msg="No images found"]

my original goal was:
 >>> montage.mArchiveList("IRAC", "1", "106.9, 1.5, gal", 11.0, 11.0, 
"irac36.tbl")

but
 >>> montage.mArchiveList("IRAC", "1", "106.9, 1.5, gal", 1.0, 1.0, 
"irac36.tbl")
is also not working.

**Comment by JohnGood** [comment]
Apologies, there was a hardware problem on our server that was causing
these list requests to fail.  It should be fixed now.

However, there is a second issue that I uncovered in checking this out.
For most datasets we maintain our own copy of the image metadata but
we have been unable to get this for the Spitzer data and are relying on
a service they maintain.  Apparently, that service is limited to a 1.5 
degree
diameter search size.  We've added a check for this in our service but it
means that you will need to build up your list piecewise.

If your interest is in the Galactic plane, we have (for another project)
built 5x5 degree plates in CAR projection from the GLIMPSE data
and would be happy to share.

- John
